Source/SwiftLintFramework/Rules/Style/VerticalWhitespaceClosingBracesRuleExamples.swift

```diff
@@ -0,0 +1,94 @@
+// swiftlint:disable:next type_name
+internal struct VerticalWhitespaceClosingBracesRuleExamples {
+    static let nonTriggeringExamples: [Example] = [
```

Comment:
No need to have the explicit type in here:
```suggestion
    static let nonTriggeringExamples = [
```

Reply:
Done.